WindowsServerDocs/remote/remote-access/directaccess/single-server-wizard/da-basic-plan-s1-infrastructure.md

```diff
@@ -110,6 +110,17 @@ In a DirectAccess deployment, DNS is required for the following:
 
     2.  If the corporate network is IPv6-based, the default address is the IPv6 address of DNS servers in the corporate network.
 
+> [!NOTE]
+> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy (NRPT).
```

Comment:
Looks more proper to include all the words which the initialism is based on:
```suggestion
> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy Table (NRPT).
```
(if there is a rationale to only name "Name Resolution Policy", you may ignore this suggestion.)